### ISSUE TITLE
check_syntax: i_where not found

### ISSUE THREAD

**Issue by ed-holland** [abaplint/abaplint#1003]
In this case, the i_where table is part of the forms input. The error is at 85.
![image](https://user-images.githubusercontent.com/53995991/85945810-775f4800-b940-11ea-8908-d708e28b76e3.png)

everywhere i_where is used, an exception is raised as it can not be found.

similar in this case:  "o_where" not found (check_syntax) on 351.
![image](https://user-images.githubusercontent.com/53995991/85945883-eb015500-b940-11ea-9f3d-3f7166b89039.png)

**Comment by larshp** [comment]
reproduced as

```abap
DATA: BEGIN OF data_foo,
        moo TYPE i,
      END OF data_foo.
FORM foo TABLES i_where STRUCTURE data_foo.
  READ TABLE i_where INDEX 1 TRANSPORTING NO FIELDS.
ENDFORM.
```

**Comment by larshp** [comment]
try again, 2.37.1 released

**Comment by ed-holland** [comment]
None where found in the rerun. Thx.
